conda-recipes/scvi/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "scvi" %}
+{% set version = "0.1.2" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "adf53e5ce8510adf5d26b5ea4e7f4ed46ac92977a7359303c87727b000a6cb54" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  host:
```

Comment:
Let's get rid of the host section --- unless you're sure it helps. I don't see other conda packages using it.

Reply:
I got the following errors if I don't include `pytest-runner` in `host` section: 
`RuntimeError: Setuptools downloading is disabled in conda build. Be sure to add all dependencies in the meta.yaml  url=https://pypi.org/simple/pytest-runner/` 
and 
`subprocess.CalledProcessError: Command '['/bin/bash', '-e', '/Users/yiningliu/miniconda3/conda-bld/scvi_1529258385675/work/conda_build.sh']' returned non-zero exit status 1.` 

I got rid of the other lines in `host`. With the newly pushed 'meta.yml`, `conda-build scvi` runs without errors.